ext/node/package_json.rs

```diff
@@ -144,6 +148,10 @@ impl PackageJson {
       }
     });
 
+    let scripts: Option<IndexMap<String, String>> = package_json
+      .get("scripts")
+      .and_then(|d| serde_json::from_value(d.to_owned()).ok());
```

Comment:
This could use a better error message (`.with_context()`)

Reply:
I'm not sure it's worth it to store an error around because I doubt it would save anyone. It would mean their "scripts" entry was something like a string or array or something.